### ISSUE TITLE
Advanced Electrolyzer

### ISSUE THREAD

**Issue by purpledragon101** [TagnumElite/ONI-Mods#3]
Good day,

With today's new update in Oxygen not Included, the mod doesn't seem to be working anymore. I love this mod and would greatly appreciate a fix. Thank you

**Comment by TagnumElite** [comment]
Yeah, I've updated the mod to be working for that update.

**Comment by purpledragon101** [comment]
I tried again today and the game is still disabling the mod unfortunately.

**Comment by TagnumElite** [comment]
You've updated both the game and mod right?

**Comment by mmmilky112** [comment]
Hi

This MOD is amazing but i dont know why it will stop working eventually
When I build it at the beginning it works perfetly, but few minutes later it stopped.
The water supply is totally the same, no related pipes (water / air) is blocked
HELP!

**Comment by mmmilky112** [comment]
More INFO about my last comment:

When it happens I will tear it down and rebuild it. And it worked again for a short while.
Now it won't work anymore no matter how many times i tear and rebuild.

**Comment by TagnumElite** [comment]
Yeah, I don't know why that could be the case. I would be the Player.log to actually see whats going on
